audb/core/info.py

```diff
@@ -52,11 +52,8 @@ def bit_depths(
 
     """
     deps = dependencies(name, version=version, cache_root=cache_root)
-    return set(
-        [
-            deps.bit_depth(file) for file in deps.media
-        ]
-    )
+    df = deps()
+    return set(df[df.type == define.DependType.MEDIA]['bit_depth'])
```

Comment:
We access columns in two different ways: `df.type` vs df[...]['bit_depth']`. I suggest to unify to:

```
df[df.type == define.DependType.MEDIA].bit_depth
```

This applies also to the other functions.

Reply:
I chnaged it accordingly.